html/browsers/the-window-object/detached-frame-property.html

```diff
@@ -0,0 +1,38 @@
+<title>Checks the value of detached subframe properties</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+execute_single_page_test_after_failing_assertion();
```

Comment:
If there has to be a big global switch (which I don't love) it should be a property passed to `setup()`

Reply:
ok